docs/plugins.md

```diff
@@ -0,0 +1,125 @@
+# Working with Plugins
```

Comment:
Let's move this to the redoc.ly/docs. I can do that.

Reply:
Yes. 👍